server/app.py

```diff
@@ -1,38 +1,45 @@
+import re
 from pathlib import Path
 
 import boto3
 import yaml
-from cryptography.hazmat.primitives import hashes, serialization
-from cryptography.hazmat.primitives.asymmetric import padding
 from flask import Flask, request
 from jose import jwt
 
+from lib.key_helper import PublicKeyManager
```

Comment:
Does the lib folder still need to be committed?

Reply:
Funny that. The big python prepackaged .gitignore excludes lib folders. I updated the .gitignore and now the files are committed.